ignition-transport2.rb

```diff
@@ -0,0 +1,35 @@
+class IgnitionTransport2 < Formula
+  desc "Transport middleware for robotics"
+  homepage "http://ignitionrobotics.org"
+  url "https://bitbucket.org/ignitionrobotics/ign-transport/get/dfbb1c1.tar.gz"
```

Comment:
you're right, it won't have the same package name. I'll change it to nate's branch